packages-exp/performance-types-exp/index.d.ts

```diff
@@ -35,6 +35,18 @@ export interface FirebasePerformance {
   dataCollectionEnabled: boolean;
 }
 
+/**
+ * This interface is for internal use only, and not exported for Firebase
+ * Performance Users.
+ */
+export interface FirebasePerformanceInternal extends FirebasePerformance {
```

Comment:
I need it for the bottom of this file (which declares the 'performance' type). If we move it to the consumer, we would need to import the src package from here (which I believe is not something we want to do).

Reply:
This file is for the users outside of performance package to be able to use it. Since the internal interface is only meant to be used within performance package and nowhere else, I prefer to have the type within the performance package and cast what we receive from component service. Although, casting is not ideal, it is preferred to having package internal mechanisms present in the public api. 
Example of this approach in js sdk: 
https://github.com/firebase/firebase-js-sdk/blob/29327b2198391a9f1e545bcd1172a4b3e12a522c/packages/performance/index.ts#L52

Reply:
Done, thanks for the info!